wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests.wiki

```diff
@@ -52,16 +53,16 @@ The [[heuristic to find places][>FindingByHeuristics]] tries a number of transla
 |page title                                                                |retrieves the current page's title (for use with 'check' verb)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
-|is visible <place>                                                        |whether specified place is visible in browser's window (used in combination with 'ensure' and 'reject')                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
-|is visible on page <place>                                                |whether specified place is visible on the page in the browser's window, scrolling may be required to actually show it (used in combination with 'ensure' and 'reject')                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|is/not visible <place> (in <container>)                                   |whether specified place is or is not visible in browser's window (used in combination with 'ensure'). Not visible will retry for <seconds before timeout> seconds if place is visible.                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Why not place 'not' in parentheses, like 'in <container>'?
I believe 'visible' will retry like 'Not visible', right?